### PR TITLE
vscode-insiders@1.57.0-insider: Fix hash

### LINKED ISSUE #5: vscode-insiders@1.57.0-insider: hash check failed
Checking hash of dl.7z ... ERROR Hash check failed!
App:         business/vscode-insiders
URL:         https://update.code.visualstudio.com/1.57.0-insider/win32-x64-archive/insider#/dl.7z
First bytes: 50 4B 03 04 14 00 00 00
Expected:    6bd485e478ed638d13e9a393bc83d78097f5157b07db6b2cc56e4595ff251f4e
Actual:      c04daa2c244785fd58d55c56f54786259b82c8eaabbd5a5ed14d9833e62a8c07

### PR DESCRIPTION
**Pull request by BotServiceAutomation**
- Closes #5

### CODE REVIEW

**Comment by BotServiceAutomation** [comment]
All changes look good.

Wait for review from human collaborators.
### vscode-insiders

- [x] Description
- [x] License
- [x] Hashes
- [x] Checkver
- [x] Autoupdate
- [x] Autoupdate Hash Extraction

**Review by alexchapar — APPROVED** [review]

**Pull request merged**
